### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by elarhadu**
In this milestone,
- [ ] I added styling to match the template given.
- [ ] I used giflow.
- [ ] I checked for linters.

### CODE REVIEW

File: src/Components/Progress.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      </div>
+      <div className="vertical-pline" />
+      <div className="chapter-status">
+        <div className="chapter-number">
+          <p>CURRENT CHAPTER</p>
+          <p>Chapter 20</p>
+          <button type="button" className="update-progress">
+            UPDATE PROGRESS
+          </button>
+        </div>
```
Comment (ichala) [inline src/Components/Progress.js]: -  to match the [required design](https://pedantic-beaver-cb923b.netlify.app/#/) please fix this issues 🙏  : 
- [ ] Update the color for the 'current chapter' title as shown in the design
- [ ] add some space between chapter 20 & the button below it
- [ ] increase the size of the percentage and change the color of 'completed' below it


Required design : 

image

your design : 
image

File: src/App.js [hunk 2]
```diff
@@ -8,9 +8,7 @@ import './App.css';
 export default function App() {
   return (
     <Router>
-
       <Navigation />
-      <h1>BOOKSTORE📖</h1>
       <Routes>
         <Route path="/" element={<Books />} />
```
Comment (ichala) [inline src/App.js:13]: -  to match the [required design](https://pedantic-beaver-cb923b.netlify.app/#/) please fix this issues 🙏  : 
- [ ] Add some padding/margins for the page content
- [ ] update the navbar links' color & remove the default text-decorations
- [ ] [OPTIONAL] I highly recommend that you use the "Montserrat",sans-serif" font to match the design

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Good Design ✔️


 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi @elarhadu 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well-written PR description ✔️ 
* All tests passed ✔️ 



------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**